ci-operator/config/openshift/knative-serving/openshift-knative-serving-master.yaml

```diff
@@ -12,6 +12,105 @@ build_root:
   project_image:
     dockerfile_path: openshift/ci-operator/build-image/Dockerfile
 canonical_go_repository: github.com/knative/serving
+binary_build_commands: make install
+test_binary_build_commands: make test-install
+images:
+- dockerfile_path: openshift/ci-operator/knative-images/activator/Dockerfile
+  from: bin
+  inputs:
+    bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/activator
+  to: activator
```

Comment:
You have a job with `--promote`, so I assume these are supposed to be part of the release. If that's the case, we probably want to have more specific image names.

Reply:
My goal is currently to have all images available in some OpenShift Docker registry where I could pull them from when running my tests. For the `knative-test-images` we won't ever want them to be shipped to customers, they're only for testing. Regarding `knative-images` (which includes the poorly named activator) I'm currently not sure when we'll want to release them but sooner or later we'll want to do that.
Can you please give me a hint how to change the config for ci-operator to achieve that?

Reply:
OK. Small update after talking to my colleagues: we don't want to release the images. They're all for testing purposes.